code/game/objects/items/handcuffs.dm

```diff
@@ -197,6 +197,14 @@
 /obj/item/restraints/handcuffs/cable/zipties/used/attack()
 	return
 
+/obj/item/restraints/handcuffs/cable/zipties/beepsky
```

Comment:
Previous review was incorrect, sorry. The cuffs being applied by beepsky are the used type which means that this typepath is pointless but the used typepath needs to be the one that exists instead.

The typepath should just be `/obj/item/restraints/handcuffs/cable/zipties/used` with a breakouttime set and no other variables set.